packages/business-rules/src/config.js

```diff
@@ -179,7 +179,7 @@ const config = {
 						},
 						reference: appeal.id
 					}),
-					saveAndReturnContinueAppeal: (appeal, baseUrl, token) => ({
+					saveAndReturnContinueAppeal: (appeal, baseUrl) => ({
```

Comment:
isn't token still required for the s+r continue email? (i.e. it is the 'code' that the user has to enter on the enter-code page)